src/rdkafka_admin.c

```diff
@@ -1647,6 +1647,55 @@ rd_kafka_error_t *rd_kafka_AdminOptions_set_match_consumer_group_states(
         return !err ? NULL : rd_kafka_error_new(err, "%s", errstr);
 }
 
+rd_kafka_error_t *rd_kafka_AdminOptions_set_match_consumer_group_types(
+    rd_kafka_AdminOptions_t *options,
+    const rd_kafka_consumer_group_type_t *consumer_group_types,
+    size_t consumer_group_types_cnt) {
+        size_t i;
+        char errstr[512];
+        rd_kafka_resp_err_t err;
+        rd_list_t *types_list = rd_list_new(0, NULL);
+        rd_list_init_int32(types_list, consumer_group_types_cnt);
+        uint64_t types_bitmask = 0;
```

Comment:
All declarations should be at the start.

Reply:
sure.

Reply:
Actually, this is in parity with all other options see states or others, the variables for list are declared inside the loop only. Rest are all at the top.

Reply:
I was talking about line 1658 and 1659. 1658 is not a declaration.